### ISSUE TITLE
Latest internal transactions do not update the wallets balance

### ISSUE THREAD

**Issue by piersandro** [blockscout/blockscout#4254]
We realized that the latest internal transactions are not recorded on Blockscout's wallet.

### Environment

* Elixir & Erlang/OTP versions (`elixir -version`): 1.11.1 & 23.3.1
* Operating System: Ubuntu 18.04
* Blockscout Version/branch: 3.7.0-beta  [previously 3.6.0-beta]

### Steps to reproduce

Latest internal transactions do not update the wallets balance
https://explorer.quadrans.io/address/0xDB92Fc7feE4DE5f6dF5d9f3d3478f6d2B22afeDC/internal-transactions

### Expected behaviour

Wallet balance aligned with the blockchain.

The balance is: 0.000000176913438435 Quadrans Coin
Latest internal transaction was: 0.056547564161310141 Quadrans Coin
The balance must be: ~0.056547741 Quadrans Coin

----------

Quadrans node uses `gqdc` binary, it is forked from `geth`. 

The Quadrans node is launched with this setup:

`/usr/local/bin/gqdc --rpc --rpcaddr 0.0.0.0 --rpcport 8545 --rpcapi debug,net,eth,shh,web3,txpool --wsapi eth,net,web3,network,debug,txpool --ws --wsaddr 0.0.0.0 --wsport 8546 --wsorigins=* --syncmode=full --gcmode=archive --rpcvhosts=*`

We upgraded the Explorer from 3.6.0-beta to 3.7.0-beta (and done a migrate) but the problem was not solved.

There is a way to force wallet balance realignment?

**Comment by vbaranov** [comment]
@piersandro I see that balance of the account, you mentioned, changed to the correct one:

<img width="471" alt="Screenshot 2021-06-08 at 21 46 35" src="https://user-images.githubusercontent.com/4341812/121240443-025ff680-c8a3-11eb-9cc5-b32979c1eafe.png">

Blockscout has an on-demand fetcher of coin balance which triggers and updates balance every time someone opens the address page. It works asynchronously in order to prevent address page loading blocking, so, the result can be seen on a second refresh.

**Comment by piersandro** [comment]
Hi @vbaranov, thank you for your reply. The wallet balance has been out of alignment for nearly 24 hours since the internal transaction. This wallet and all the others that received the mining reward from the network.
I reloaded the page (and restarted the fetcher) a lot of time before opened this issue. What can I check to understand why there was a misalignment for 24 hours?

**Comment by vbaranov** [comment]
@piersandro ah, yes, forgot to mention. An on-demand coin balance fetcher has a threshold to trigger. It is configured here https://github.com/blockscout/blockscout/blob/a9d4351c770098e2c17fc46836b4c979534bab25/apps/indexer/lib/indexer/fetcher/coin_balance_on_demand.ex#L10 and by default, it is 24 hrs. If you want, to trigger this fetcher more often, you should reduce the value for this module attribute.

**Comment by piersandro** [comment]
hi @vbaranov, I'm sorry for the delay. The issue returned randomly.
For example one user reported me his wallet two days ago but the balance is not aligned (the last internal tx is not in the balance): https://explorer.quadrans.io/address/0x23E7FEca48Ee298315dD86A42ed36826D4e5921f/internal-transactions

At the same time the Fetching tokens animation never ends.

Today I noticed that the transactions are halted to yesterday. I tried to restart the service with no success. 

I have no strange errors in the logs so I don't understand what causing it.
